src/_data/defaults.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	theme: 'default',
+	showLatestMessages: 100,
+	DEMO: false,
+	showCommands: false,
+	channel: 'showmychat',
+	hideMessagesFrom: '',
+};
```

Comment:
Can we put `channel` and `DEMO` at the top of this and then alphabetize the remaining attributes?

Reply:
Done!
Is this a wanted pattern, to sort keys alphbetically?
Pretty sure that can be checked against by flipping a config option for ESLint